packages/runtime/src/helpers/edge.ts

```diff
@@ -485,10 +490,11 @@ export const writeEdgeFunctions = async ({
       join('.netlify', 'functions-internal', IMAGE_FUNCTION_NAME, 'imageconfig.json'),
       join(edgeFunctionDir, 'imageconfig.json'),
     )
+
     manifest.functions.push({
       function: 'ipx',
       name: 'next/image handler',
-      path: nextConfig.images.path || '/_next/image',
+      path: nextConfig.images.path ? sanitizeEdgePath(nextConfig.images.path) : '/_next/image',
```

Comment:
It'd be good to either have a unit test checking that it gets sanitized when needed or we could update a demo site with a full images path in a Next config to validate the fix.